src/inpod.rs

```diff
@@ -66,15 +66,11 @@ impl WorkloadUid {
     }
 }
 
-#[derive(Debug, Clone)]
-struct WorkloadInfo {
-    workload_uid: WorkloadUid,
-    pod_info: Option<istio::zds::PodInfo>,
-}
 #[derive(Debug)]
 pub struct WorkloadData {
     netns: std::os::fd::OwnedFd,
-    info: WorkloadInfo,
+    workload_uid: WorkloadUid,
```

Comment:
This was in response to johns question here: https://github.com/istio/ztunnel/pull/747#discussion_r1440759395
The type serves as "documentation" that this is a uid.

Reply:
I'd rather just do that with the normal function comment/variable name since this is a bit tedious in practice and just results in a lot of arguably useless type conversion, but will defer if he feels strongly.